controller/list_controller.php

```diff
@@ -63,13 +63,12 @@ public function ideas_list($sort)
 		$ideas = $this->ideas->get_ideas($this->config['posts_per_page'], $sort, $sort_direction, $status, $where, $start);
 		$this->assign_template_block_vars('ideas', $ideas);
 
-		$statuses = $this->ideas->get_statuses();
-		foreach ($statuses as $status_row)
+		foreach (ideas::$statuses as $statuses_name => $statuses_id)
```

Comment:
Yes, the array keys are the names. Since we usually want the numbers, and get them by doing, `$statuses['NEW']`